internal/application/app_frontend.go

```diff
@@ -80,9 +81,15 @@ func (a *App) StartFrontend(ctx context.Context) error {
 	// proxy requests to the next subprocess
 	proxy := httputil.NewSingleHostReverseProxy(feUrl)
 
-	a.mux.HandleFunc("/", func(writer http.ResponseWriter, request *http.Request) {
-		proxy.ServeHTTP(writer, request)
-	})
+	handler := func(writer http.ResponseWriter, request *http.Request) {
```

Comment:
@pnwx not sure if this is the best way to handle serving both the marketing site and app. I tried a few different things but this was the one thing that works.